code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -246,7 +251,7 @@
 
 /datum/reagent/fuel/unholywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	if(method == TOUCH || method == VAPOR)
-		M.reagents.add_reagent("unholywater", (reac_volume/4))
+		M.reagents.add_reagent(src.id,reac_volume/5)
```

Comment:
src.

Reply:
Ugh what, I thought I changed this far more... this doesn't fix anything.